# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/apmd/makefile-fixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2005-12-20 12:13:15.000000000 +0100
+++ ./Makefile	2005-12-20 12:13:49.000000000 +0100
@@ -28,7 +28,10 @@
 .SUFFIXES:
 
 OBJS=apmlib.o
-EXES=apm apmd xapm apmsleep
+EXES=apm apmd apmsleep
+ifndef SKIP_XAPM
+  EXES += xapm
+endif
 HEADERS=apm.h
 
 PREFIX=/usr
@@ -37,8 +40,7 @@
 
 CC=gcc
 CFLAGS=-O -g
-XTRACFLAGS=-Wall -pipe -I. -I/usr/src/linux/include \
-		-I/usr/src/linux-2.2/include -I /usr/src/linux-2.0/include \
+XTRACFLAGS=-Wall -pipe -I. -I/usr/include \
 		-DVERSION=\"$(VERSION)\" \
 		-DAPMD_PROXY_NAME=\"$(APMD_PROXY_DIR)/apmd_proxy\"
 LDFLAGS=
@@ -92,10 +94,12 @@
 	install        on_ac_power     ${PREFIX}/bin
 	install -m 644 apm.1           ${MANDIR}/man1
 	install        apmd            ${PREFIX}/sbin
-	install -b -Vt apmd_proxy      ${APMD_PROXY_DIR}
+	install -b     apmd_proxy      ${APMD_PROXY_DIR}
 	install -m 644 apmd.8          ${MANDIR}/man8
+ifndef SKIP_XAPM
 	install        xapm            ${PREFIX}/bin
 	install -m 644 xapm.1          ${MANDIR}/man1/xapm.1
+endif
 	install        apmsleep	       ${PREFIX}/bin
 	install -m 644 apmsleep.1      ${MANDIR}/man1
 	install -m 644 apmsleep.1.fr   ${MANDIR}/fr/man1
